lib/smart_proxy_salt/cli.rb

```diff
@@ -20,52 +20,51 @@ def autosign_key_file
 
         def autosign_create_hostname(hostname)
           if append_value_to_file(autosign_file, hostname)
-            { message: 'Added hostname successfully.' }
+            { :message => 'Added hostname successfully.' }
```

Comment:
It's surprising that `Style/HashSyntax` does enforce this style. I'd suggest to use the style that was used previously. It's often considered the more modern approach. Perhaps disable the cop, given there's already a mixed usage?

Reply:
I tried with disabling the cop and moving back to the previous changes.